python/cugraph/cugraph/tests/mg/test_mg_betweenness_centrality.py

```diff
@@ -0,0 +1,215 @@
+def input_combo(request):
+    """
+    Simply return the current combination of params as a dictionary for use in
+    tests or other parameterized fixtures.
+    """
+    parameters = dict(
+        zip(
+            ("graph_file", "has_vertex_list", "normalized", "endpoints", "directed"),
+            request.param,
+        )
+    )
+
+    return parameters
+
+
+@pytest.fixture(scope="module")
+def input_expected_output(input_combo):
+    """
+    This fixture returns the inputs and expected results from the
+    betweenness_centrality algo based on cuGraph betweenness_centrality) which can
+    be used for validation.
+    """
+
+    input_data_path = input_combo["graph_file"]
+    directed = input_combo["directed"]
+    normalized = input_combo["normalized"]
+    endpoints = input_combo["endpoints"]
+    vertex_list = None
+
+    input_combo["vertex_list"] = vertex_list
+    G = utils.generate_cugraph_graph_from_file(input_data_path, directed=directed)
+
+    if input_combo["has_vertex_list"]:
+        # Sample vertices from the graph
+        k = random.randint(1, 4)
+        vertex_list = G.nodes().compute().sample(k).reset_index(drop=True)
+
+    sg_cugraph_bc = cugraph.betweenness_centrality(
+        G, vertex_list=vertex_list, normalized=normalized, endpoints=endpoints
+    )
+    # Save the results back to the input_combo dictionary to prevent redundant
+    # cuGraph runs. Other tests using the input_combo fixture will look for
+    # them, and if not present they will have to re-run the same cuGraph call.
+    sg_cugraph_bc = sg_cugraph_bc.sort_values("vertex").reset_index(drop=True)
+
+    input_combo["sg_cugraph_results"] = sg_cugraph_bc
+    chunksize = dcg.get_chunksize(input_data_path)
+    ddf = dask_cudf.read_csv(
+        input_data_path,
+        chunksize=chunksize,
+        delimiter=" ",
+        names=["src", "dst", "value"],
+        dtype=["int32", "int32", "float32"],
+    )
+
+    dg = cugraph.Graph(directed=directed)
+    dg.from_dask_cudf_edgelist(
+        ddf,
+        source="src",
+        destination="dst",
+        edge_attr="value",
+        renumber=True,
+        legacy_renum_only=True,
+        store_transposed=True,
+    )
+
+    input_combo["MGGraph"] = dg
+
+    return input_combo
+
+
+# =============================================================================
+# Tests
+# =============================================================================
+
+
+# @pytest.mark.skipif(
+#    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+# )
+def test_dask_betweenness_centrality(dask_client, benchmark, input_expected_output):
+
+    dg = input_expected_output["MGGraph"]
+    vertex_list = input_expected_output["vertex_list"]
+    endpoints = input_expected_output["endpoints"]
+    normalized = input_expected_output["normalized"]
+
+    result_betweenness_centrality = benchmark(
+        dcg.betweenness_centrality,
+        dg,
+        vertex_list=vertex_list,
```

Comment:
Lets add a test here for various vertex_list types.

Reply:
Done